Doc/library/unittest.rst

```diff
@@ -2019,16 +2033,25 @@ Loading and running tests
 
       .. versionadded:: 3.4
 
+   .. method:: addDuration(test, elapsed)
+
+      Called when the test case finishes. *elapsed* is the time represented
+      in seconds.
+
+      .. versionadded:: 3.8
 
-.. class:: TextTestResult(stream, descriptions, verbosity)
+.. class:: TextTestResult(stream, descriptions, verbosity, *, durations=None)
 
    A concrete implementation of :class:`TestResult` used by the
-   :class:`TextTestRunner`.
+   :class:`TextTestRunner`. Subclasses should accept ``**kwargs`` to ensure
```

Comment:
Perhaps create a `versionchanged:: 3.8` directive and add this note : "Subclasses should accept ``**kwargs`` to ensure compatibility as the interface changes." . When docs are merged and rendered this might read like 3.7 also had this promise to accept kwargs and might be better to make this distinctive.

Reply:
Done